pkg/controller/account/account_controller.go

```diff
@@ -253,12 +253,8 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 
 			if !accountHasAwsAccountID(currentAcctInstance) {
 				// before doing anything make sure we are not over the limit if we are just error
-				if totalaccountwatcher.TotalAccountWatcher.Total == 0 {
-					reqLogger.Error(awsv1alpha1.ErrAccountWatcherNoTotal, "AccountWatcher has not run successfully yet")
-					return reconcile.Result{}, awsv1alpha1.ErrAccountWatcherNoTotal
-				}
-				if ok, _ := checkAWSAccountsLimitReached(r, reqLogger, totalaccountwatcher.TotalAccountWatcher.Total); ok {
-					reqLogger.Error(awsv1alpha1.ErrAwsAccountLimitExceeded, "AWS Account limit reached", "Account Total", totalaccountwatcher.TotalAccountWatcher.Total)
+				if totalaccountwatcher.TotalAccountWatcher.AccountsCanBeCreated() {
```

Comment:
Isn't this backwards?
```suggestion
				if !totalaccountwatcher.TotalAccountWatcher.AccountsCanBeCreated() {
```